### ISSUE TITLE
Improvements to navigation in Restricted mode of Spiral

### ISSUE THREAD

**Issue by ksraj123** [llaske/sugarizer#657]
Spiral in homeview has a Restricted mode for small screen size devices. Buttons for `showNextRestrictedList` and `showPreviousRestrictedList` are there in the Restricted Mode to navigate through the list and locate the activity of choice, but it leads to bad user experience as several clicks might be needed to locate any activity. I feel having the ability to scroll the spiral will improve the user experience specially for touch based small screen size devices like mobile phones.

This is what the Restricted mode currently looks like - 
![before restricted mode](https://user-images.githubusercontent.com/39027928/75891042-3a34ee80-5e55-11ea-943f-cc94860c736e.gif)

I am currently working on this issue. 

There might be several bugs or possible improvements in the PR associated with this issue right now. Opening this issue and the associated PR to know if this is a wanted change? Requesting feedback from the community.

**Comment by llaske** [comment]
Hmmm. It could be interesting to provide another way to navigate but I think scrolling is neither practical, neither intuitive. May be a touch in left direction to go left and a touch in the right direction to go right? Like in Falabracman game.

**Comment by ksraj123** [comment]
@llaske Yeah, I get your point. Its meant to be used by children and scrolling is not really intuitive here. A half-baked prototype version of scrolling is implemented in #658 . Actually the comments in the code asked for a way to integrate scrolling on the circle in restricted mode. https://github.com/llaske/sugarizer/blob/dd3c58d56981e4dfa6d94601b95d7117a3bc9241/js/homeview.js#L332 
Working on a different approach now, will send a PR soon.

**Comment by llaske** [comment]
I've tested https://github.com/llaske/sugarizer/pull/658. Thanks for that but as I said, using scrolling is not intuitive and is not practical.

The comment you mention is just related to scrolling using `showNextRestrictedList` and `showPreviousRestrictedList` buttons.

**Comment by llaske** [comment]
Closing issue due to lack of activity
